src/viz/validation.ts

```diff
@@ -50,19 +50,19 @@ No files have been created.
 };
 
 export const parseBucketName = (bucketPath: string): string | undefined => {
-  const matches = bucketPath.match(/(gs:\/\/[^/\s]+)/);
+  const matches = bucketPath.match(/(gs:\/\/[-a-zA-Z]+)\/?([-a-zA-Z]+\/?)*$/);
```

Comment:
Use `(?: ... )` instead of `( ... )` to create a non-capturing group, so that your match's index is still `0`. This is more semantically correct; a `(?: ... )` group says "I want to group this for repetition reasons but I'm not interested in extracting the result afterwards.